packages/land-sale/deadlines.ts

```diff
@@ -0,0 +1,57 @@
+const deadlines: {[sector: number]: number} = {
```

Comment:
Can we move those files in a folder `scripts` or `utils`

Reply:
Absolutely, I think sine those contain some information I've moved into `data`